students/FyfyNguyen/session03/mailroom.py

```diff
@@ -1,35 +1,103 @@
-#!/usr/bin python3
+#!/usr/bin/env python3
 
 """
-mailroom assignment
+Mailroom 1
 """
+import sys
+
+donors = [("Robyn Rihanna", [100, 200, 300]),
+          ("Ariana Grande", [2250, 4000, 1000]),
+          ("Beyonce Carter-Knowles", [150000, 3500, 25000]),
+          ("Aubrey Drake Graham", [15000, 5500.25, 1200]),
+          ("Justin Bieber", [2500, 250, 750.50])
+          ]
+
+
+def print_donors():
+    for donor in donors:
+        print(donor[0])
+
+
+def find_donor(name_entered):
+    for donor in donors:
+        if name_entered.strip().lower() == donor[0].lower():
+            return donor
+    return None
+
+
+def send_email(name_entered, donation):
+    print(
+        """Dear {},
```

Comment:
yes, tripple quotes is perfect for this sort of thing.

But as you noticed, it does require to to not indent, which is a bit ugly.

The the `dedent` function in the `textwrap` module can help here:

https://docs.python.org/3.1/library/textwrap.html#textwrap.dedent